src/front/ticketer/lib/pages/page_organizer_register.dart

```diff
@@ -0,0 +1,316 @@
+      controller: _phone,
+      decoration: const InputDecoration(
+          labelText: "Phone", hintText: 'Enter phone number'),
+      validator: (value) {
+        if (value == null || value.isEmpty) {
+          return "Please enter phone number";
+        }
+        return null;
+      },
+    );
+  }
+
+  Widget _submitButton() {
+    return Container(
+        margin: const EdgeInsets.only(top: 15.0),
+        child: ElevatedButton(
+          onPressed: () => submitOrganizerData(),
+          child: const Text("Submit"),
+        ));
+  }
+
+  Widget _getRegisterContent() {
+    return Form(
+      key: _formKey,
+      child: SingleChildScrollView(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            _titleBanner(),
+            _companyNameEntryField(),
+            _companyCityEntryField(),
+            _companyZipCodeEntryField(),
+            _companyStreetEntryField(),
+            _companyDisplayNameEntryField(),
+            _taxTypeRow(),
+            _companyPhoneEntryField(),
+            _submitButton(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Text _titleBanner() {
+    return const Text(
+      "Join us as an organizer",
+      style: TextStyle(
+        fontSize: 22,
+        fontWeight: FontWeight.w600,
+        color: Colors.blueAccent,
+      ),
+    );
+  }
+
+  Row _taxTypeRow() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: <Widget>[
+        Flexible(
+          flex: 3,
+          child: TaxTypeDropdown(setTaxType: setTaxType),
+        ),
+        const Spacer(),
+        Flexible(
+          flex: 7,
+          child: _companyTaxIdEntryField(),
+        ),
+      ],
+    );
+  }
+
+  Future<void> submitOrganizerData() async {
+    if (_formKey.currentState!.validate()) {
+      Organizer organizer = Organizer(
+          _companyName.text,
+          '${_addressStreet.text}/${_addressZip.text}/${_addressCity.text}',
+          taxType,
+          _taxId.text,
+          _displayName.text,
+          credentials.email,
+          credentials.password,
+          _phone.text);
+      sendOrganizerRegistrationRequest(organizer);
+
+      showDialog(
+        context: context,
+        builder: (context) {
+          return AlertDialog(
+            title: const Text("Thank you"),
+            content: const Text("Administrators have received your form. "
+                "We will try to verify your data as soon as possible."),
+            actions: [
+              ElevatedButton(
+                onPressed: () => Navigator.pop(context),
+                child: const Text('OK'),
+              ),
+            ],
+          );
+        },
+      );
+    }
+  }
+
+  void sendOrganizerRegistrationRequest(Organizer organizer) async {
+    String url = "localhost:5166";
```

Comment:
Shouldn't you move to some global env? It will potentially change in the future and will be totally different when we start deploying app

Reply:
I've tried to, it was causing me some trouble, but maybe it was connected to another bug I resolved. I'll try it out once again.

Reply:
Done

Reply:
This might be a good idea if not for Flutter not encrypting .env files. Will have to look into it for sure tho